commit 3676e278dbe2111d2799a4d2dc023f20e9cd6820
Author: Alex Bennée <alex.bennee@linaro.org>
Date:   Tue Sep 24 20:43:56 2024 +0100

    config/targets: update aarch64_be-linux-user gdb XML list

    Attempting to run the binary asserts when it can't find the XML entry.
    We can fix it so we don't although I suspect other stuff is broken.

    Fixes: https://gitlab.com/qemu-project/qemu/-/issues/2580
    Signed-off-by: Alex Bennée <alex.bennee@linaro.org>

diff --git a/configs/targets/aarch64_be-linux-user.mak b/configs/targets/aarch64_be-linux-user.mak
index acb5620cdb..14623ac076 100644
--- a/configs/targets/aarch64_be-linux-user.mak
+++ b/configs/targets/aarch64_be-linux-user.mak
@@ -1,7 +1,7 @@
 TARGET_ARCH=aarch64
 TARGET_BASE_ARCH=arm
 TARGET_BIG_ENDIAN=y
-TARGET_XML_FILES= gdb-xml/aarch64-core.xml gdb-xml/aarch64-fpu.xml gdb-xml/aarch64-pauth.xml
+TARGET_XML_FILES= gdb-xml/aarch64-core.xml gdb-xml/aarch64-fpu.xml gdb-xml/aarch64-pauth.xml gdb-xml/aarch64-mte.xml
 TARGET_HAS_BFLT=y
 CONFIG_SEMIHOSTING=y
 CONFIG_ARM_COMPATIBLE_SEMIHOSTING=y
